test/e2e/framework/pod/wait_test.go

```diff
@@ -84,7 +84,7 @@ INFO: Ignoring NotFound error while getting pod default/no-such-pod
 INFO: Unexpected error: 
     <*fmt.wrapError>: {
         msg: "error while waiting for pod default/no-such-pod to be running: pods \"no-such-pod\" not found",
-        err: {
+        err: <*errors.StatusError>{
```

Comment:
not sure which change from `gomega` import this, seems like this one: https://github.com/onsi/gomega/pull/408? @onsi

Reply:
Yes, looks like the gomega version bump causes this. This is fine.